ext/standard/mail.c

```diff
@@ -281,6 +320,11 @@ PHPAPI int php_mail(char *to, char *subject, char *message, char *headers, char
 		efree(f);
 	}
 
+	if (hdr && php_mail_detect_multiple_crlf(hdr)) {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Multiple or malformed newlines found in additional_header");
+		MAIL_RET(0);
```

Comment:
```
		$headers .= '--' . $separator . $eol;
		$headers .= "Content-Type: application/pdf; name=\"" . $filename . "\"" . $eol;
		$headers .= 'Content-Transfer-Encoding: base64' . $eol;
		$headers .= "Content-Disposition: attachment; filename=\"" . $filename . "\"" . $eol . $eol;
		$headers .= chunk_split(base64_encode($invoicePDF));
		$headers .= '--' . $separator . '--';
```
Change
```
		$headers .= "Content-Disposition: attachment; filename=\"" . $filename . "\"" . $eol . $eol;
```
into
```
		$headers .= "Content-Disposition: attachment; filename=\"" . $filename . "\"" . $eol;
```
and you end up with an empty attachment.

Tested with [mandrill](https://www.mandrill.com/)